### PR TITLE
changes in input value with edge cases

### PR DESCRIPTION
**Pull request by mehulsatardekar**
made a changes in input value with min and max attributes as well as added a edge case if some user tries to add number through input bigger than expected value will show an error msg through alert box

### CODE REVIEW

File: templates/dice_page.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 <body>
   <div id="input">
   <label>Enter the number of sides of the dice</label>
-  <input type="number" value="6" id="dicesides"></input>
+  <input type="number" value="6" min="0" max="6" id="dicesides"></input>
```
Comment (srujandeshpande) [inline templates/dice_page.html]: Min looks good. It would be great if max can be left open, that way larger dice can also be rolled

**Review by srujandeshpande — CHANGES_REQUESTED** [review]
Please remove the max

**Review by mehulsatardekar — COMMENTED** [review]
made a changes as you suggested.

**Review by srujandeshpande — APPROVED** [review]
Looks good!

**Comment by srujandeshpande** [comment]
Thanks @mehulsatardekar looks great!

**Pull request merged**